src/app/public/plugin-resources/code-examples/split-view/page-bound/split-view-page-bound-demo.component.ts

```diff
@@ -0,0 +1,227 @@
+  templateUrl: './split-view-page-bound-demo.component.html',
+  styleUrls: ['./split-view-page-bound-demo.component.scss']
+})
+export class SplitViewPageBoundDemoComponent implements OnInit {
+
+  public set activeIndex(value: number) {
+    this._activeIndex = value;
+    this.activeRecord = this.items[this._activeIndex];
+    this.loadFormGroup(this.activeRecord);
+  }
+
+  public get activeIndex(): number {
+    return this._activeIndex;
+  }
+
+  public offsetTop: number;
+
+  public activeRecord: any;
+
+  public items = [
+    {
+      id: 1,
+      amount: 73.19,
+      date: '5/13/2020',
+      vendor: 'amazon.com',
+      receiptImage: 'amzn-office-supply-order-5-13-19.png',
+      approvedAmount: 73.19,
+      comments: ''
+    },
+    {
+      id: 2,
+      amount: 214.12,
+      date: '5/14/2020',
+      vendor: 'Office Max',
+      receiptImage: 'office-max-order.png',
+      approvedAmount: 214.12,
+      comments: ''
+    },
+    {
+      id: 3,
+      amount: 29.99,
+      date: '5/14/2020',
+      vendor: 'amazon.com',
+      receiptImage: 'amzn-office-supply-order-5-14-19.png',
+      approvedAmount: 29.99,
+      comments: ''
+    },
+    {
+      id: 4,
+      amount: 1500,
+      date: '5/15/2020',
+      vendor: 'Fresh Catering, LLC',
+      receiptImage: 'fresh-catering-llc-order.png',
+      approvedAmount: 1500,
+      comments: ''
+    },
+    {
+      id: 5,
+      amount: 456.24,
+      date: '5/16/2020',
+      vendor: 'Kerblam',
+      receiptImage: 'kerblam-delivery-order.png',
+      approvedAmount: 456.24,
+      comments: ''
+    },
+    {
+      id: 6,
+      amount: 62.37,
+      date: '5/16/2020',
+      vendor: 'Staples',
+      receiptImage: 'staples-paper-bulk-order.png',
+      approvedAmount: 62.37,
+      comments: ''
+    },
+    {
+      id: 7,
+      amount: 51.84,
+      date: '5/17/2020',
+      vendor: 'amazon.com',
+      receiptImage: 'amzn-office-supply-order-5-17-19.png',
+      approvedAmount: 51.84,
+      comments: ''
+    },
+    {
+      id: 8,
+      amount: 92.55,
+      date: '5/18/2020',
+      vendor: 'Lowes',
+      receiptImage: 'lowes-order.png',
+      approvedAmount: 0.00,
+      comments: ''
+    },
+    {
+      id: 9,
+      amount: 38.29,
+      date: '5/18/2020',
+      vendor: 'Papa Johns',
+      receiptImage: 'papa-johns-order.png',
+      approvedAmount: 38.29,
+      comments: ''
+    }
+  ];
+
+  public listWidth: number;
+
+  public splitViewDemoForm: FormGroup;
+
+  public splitViewStream = new Subject<SkySplitViewMessage>();
+
+  public unapprovedTransaction: boolean = true;
+
+  private _activeIndex = 0;
+
+  constructor(
+    public confirmService: SkyConfirmService
+  ) {
+    // Start with the first item selected.
+    this.activeIndex = 0;
+  }
+
+  public ngOnInit(): void {
+    const omnibarIframe = document.querySelector('.sky-omnibar-iframe');
+    const skyBuilderHeader = document.getElementById('skypages-header');
```

Comment:
Instead of putting these calculations in the demo, I would rename the property to `pageFlexOffsetTop` and just set it to `0`. Adding a comment above the statement should be enough to communicate the idea.

```
// Make any adjustments to the page's offset top to accommodate fixed-positioned elements,
// such as a "sticky" navbar, etc.
this.pageFlexOffsetTop = 0;
```

Reply:
Done. I liked having the calculations in there because it showed the consumer how to do what would be a common thing; however, I also get that technically this is the visual library and external consumers shouldn't necessarily know about omnibar and the builder internals.